core/app/modules/kit-library/connect/kit-library.php

```diff
@@ -0,0 +1,41 @@
+<?php
+namespace Elementor\Core\App\Modules\KitLibrary\Connect;
+
+use Elementor\Core\Common\Modules\Connect\Apps\Library;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly
+}
+
+class Kit_Library extends Library {
+	// TODO: NEED TO BE CHANGED!
+	const DEFAULT_BASE_ENDPOINT = 'https://kits.dev.elementor.red';
```

Comment:
Just wonder why its 'need to be changed'?

Reply:
Should be changed to the production URL when it will deploy to production.